_data/helen8297/helen8297.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Helen
+institution: School of Code Bootcamp, Birmingham.
+profile_pic: graduation/_data/helen8297/helen.JPG
```

Comment:
```suggestion
profile_pic: helen.JPG
```
We added new instructions to make it more clear on what we look for here. You can accept this suggestion to automagically modify it. Sorry about it 🙌